sadcowpng/sharedspaces/forms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms
+
+# Tuple list used for Multiple Choice Field
```

Comment:
super super nit: My old host told me that when commenting, you don't need to mention the data structure explicitly. Instead, saying that this represents a multiple choice field mapping for question x is a bit better. Again, super nit but I always thought it was a nice thing to have in mind to write clearer comments.

Reply:
Updated